### DISCUSSION TITLE
Can not find City-Mind on hacs

### DISCUSSION THREAD

**Discussion by yanvar** [maorcc/citymind_water_meter#54] [General]
Hi,

I am trying to download and install the integration
Can not find in hacs the integration nor the repository when searching for "City-Mind water meter"
Is it still allive?

**Comment by elad-bar** [comment]
Pls add it as custom repository

Thanks

**Comment by yanvar** [comment]
Hi Elad,

Nice to e-meet you and thanks for the fast response.
I have hacs installed on my home assistant and for some reason I can not
see the citymind integration.
I also tried to download the src and place it in
/config/custome_components/ then restart HA, but I still can not see the
integration in both HACS and the integration menu.

Probably I am doing something wrong...
What do you mean by custom repository?

Yaniv

**Comment by elad-bar** [comment]
Hi Yaniv,

within HACS --> Integration click on the 3 dots menu, you will see an option of Custom repositories.
in the form fill in:
Repository: maorcc/citymind_water_meter
Category: Integration

click on Add.

when you will search for the integration, you will find it.

**Comment by yanvar** [comment]
Thanks for the reply,
I managed to set it up

Thank you,
Yaniv

**Reply by maorcc** [reply]
Hi @yanvar,
Notice that we also published a variant of this integration as an official Home Assistant integration.
It can be found here: https://www.home-assistant.io/integrations/rympro/
The official HA integration provides the Water Meter sensor but it comes with fewer statistics sensors. Your choice.

"Choice is good!" 👍🏼
